src/Core/Framework/DataAbstractionLayer/Search/Parser/SqlQueryParser.php

```diff
@@ -235,6 +235,10 @@ private function parseEqualsFilter(EqualsFilter $query, EntityDefinition $defini
         $result = new ParseResult();
 
         if ($field instanceof ListField) {
+            if ($query->getValue() === null) {
```

Comment:
Made an attempt to write a test for this. Please let me know if there is a better way to do it.

Should I test the parse function rather than using reflection to test the private parseEqualsFilter ?